### PR TITLE
Updates to the codestyle

### PR DESCRIPTION
**Pull request by akuleshov7**
### What's done:
Regenerated the codestyle

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Now diKTat was already added to the lists of [static analysis tools](https://git
 
 |  |  |  |  |
 | --- | --- | --- | --- |
-|[DiKTat codestyle](info/guide/diktat-coding-convention.md)|[Supported Rules](info/available-rules.md) | [Examples of Usage](https://github.com/akuleshov7/diktat-examples) | [Online Demo](https://ktlint-demo.herokuapp.com) |
+|[DiKTat codestyle](info/guide/diktat-coding-convention.md)|[Supported Rules](info/available-rules.md) | [Examples of Usage](https://github.com/akuleshov7/diktat-examples) | [Online Demo](https://ktlint-demo.herokuapp.com) | [Whitepaper](wp/wp.pdf) |
```
Comment (petertrr) [inline README.md:31]: Have you decided to upload pdf to repo after all? Or we could add link to README in #655, as we planned, with link to gh releases.

File: info/guide/diktat-coding-convention.md [hunk 2]
```diff
@@ -107,6 +108,8 @@ I [Preface](#c0)
     * [6.4.2 Objects should be used for Stateless Interfaces](#r6.4.2) 
 
 
+# Huawei Coding Style Guide
```
Comment (petertrr) [inline info/guide/diktat-coding-convention.md:111]: Not diktat?

File: info/guide/guide-TOC.md [hunk 3]
```diff
@@ -1,3 +1,4 @@
+# Diktat Coding Style Guide, v.0.0.1
```
Comment (petertrr) [inline info/guide/guide-TOC.md:1]: How will we track these versions?

**Review by petertrr — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (akuleshov7) [reply README.md:31]: yes, after all

File: info/guide/guide-TOC.md (hunk 3 above)
Comment (akuleshov7) [reply info/guide/guide-TOC.md:1]: I actually think that it is enough to have the version here

**Review by petertrr — APPROVED** [review]

**Pull request merged**